actual/api/__init__.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import annotations
 
 import datetime
+from enum import verify
```

Comment:
Was this imported by mistake?

Reply:
yup, you'r right, sorry about that, fixed :pray: